### PR TITLE
Add Jupyterlite

### PR DESCRIPTION
**Pull request by paddymul**
[Jupyter LIte](https://github.com/jupyterlite) Allows you to run an entire jupyter instance in your browser via Pyodide and Web Assembly.  This lets users start running notebooks instantly (once they download 40-60Megs of python distribution) without signing up for any services.

From my deployment, you can try it out here:
https://paddymul.github.io/pandas-cookbook/lab/index.html

### CODE REVIEW

File: cookbook/Chapter 2 - Selecting data & finding the most common complaint type.ipynb [hunk 1]
```diff
@@ -36,7 +36,7 @@
    "outputs": [],
    "source": [
     "# because of mixed types we specify dtype to prevent any errors\n",
-    "complaints = pd.read_csv('../data/311-service-requests.csv', dtype='unicode')"
```
Comment (paddymul) [inline cookbook/Chapter 2 - Selecting data & finding the most common complaint type.ipynb:39]: Jupyterlite wants the entire tree under one directory, so I moved data under cookbook.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
 Pandas cookbook
 ===============
 
-[![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/jvns/pandas-cookbook/master)
```
Comment (paddymul) [inline README.md:4]: Binder has fallen out of favor in the jupyter community generally because became very unreliable after google stopped supporting it

**Comment by jvns** [comment]
wow I'd never heard of Jupyterlite, this is awesome. Thanks!

**Comment by jvns** [comment]
The Github Action you set up looks like it's working great! It's now at https://jvns.github.io/pandas-cookbook/lab/index.html

I removed the "Running Jupyterlite locally" instructions from the README because I couldn't get them to work (I didn't understand how to install the `jupyter lite` subcommand), but I'd be very happy to put them back if there are instructions for how to get them working.

**Pull request merged**